src/components/DataProvider/index.js

```diff
@@ -13,6 +13,11 @@ export const calculateDomainFromData = (
   minAccessor = null,
   maxAccessor = null
 ) => {
+  // if there is no data, hard code the domain
+  if (!data || !data.length) {
+    return [-0.25, 0.25];
```

Comment:
I don't really know what we should default to here

Reply:
Seems as good as anything.